cocos/2d/assets/sprite-frame.ts

```diff
@@ -537,6 +589,28 @@ export class SpriteFrame extends Asset {
 
     protected _packable = true;
 
+    protected _pixelsToUnit = 100;
+
+    protected _pivot = new Vec2(0.5, 0.5); // center
+
+    // Todo: Some features need add
+    protected _meshType = MeshType.RECT;
+    protected _extrude = 0; // when polygon type use
+    protected _customOutLine = [];// MayBe later
+    // Here you can generate polygons by polygon-separator, expecting the generated polygons to be pixel-standard vertex arrays,
+    // and save this array in the file
+    // that is, the mesh information of the original image
+    // (two-dimensional mesh information, which needs to be serialized and is the basis for the polygon mesh generation)
+    // In addition to the vertex array, a mesh should be generated based on the conditions
+    // i.e., the mesh information generated by combining the above five conditions, which needs to be serialized
+    // and at runtime, the actual mesh used is the generated mesh (static mesh)
+    // (updated after attribute value changes in the editor, adjusting vertices/re-generation)
+
+    // Mesh api
+    protected declare _mesh: Mesh;
```

Comment:
If _mesh could be absent, this shouldn't be using declare, declare means it must be assigned during construction.

Also the type should be `Mesh | null`